lib/pdk/util/version.rb

```diff
@@ -23,9 +23,12 @@ def self.pkg_sha
       end
 
       def self.git_ref
-        ref_result = PDK::CLI::Exec.git('--git-dir', File.join(File.expand_path('../../..', File.dirname(__FILE__)), '.git'), 'describe', '--all', '--long')
+        source_git_dir = File.join(File.expand_path('../../..', File.dirname(__FILE__)), '.git')
 
-        ref_result[:stdout].strip if ref_result[:exit_code].zero?
+        return '' unless File.directory?(source_git_dir)
```

Comment:
Can we return nil here instead of an empty string?

Reply:
yes